### ISSUE TITLE
how we autologin in moodle for odoo user

### ISSUE THREAD

**Issue by kartikdanidhariya** [EE/moodle_auth_odoo#2]
hi,
moodle_auth_odoo is work perfectly after your support in my previous issue #1 
but now i want to auto login in moodle for odoo users
it means when any odoo users login into odoo so its automatically login also in moodle
please give your valuable suggestions for this, 
thanks in advance!!

**Comment by ludwiktrammer** [comment]
Unfortunately this plugin doesn't support this. It would be much harder to do.

**Comment by kartikdanidhariya** [comment]
should we apply this from handle session for users.

**Comment by ludwiktrammer** [comment]
I don't understand the question. Can you rephrase it?

**Comment by kartikdanidhariya** [comment]
My question is :

When user logins in the ODOO at that time that user gets automatically logged in MOODLE also without any user inputs.
